### PR TITLE
Add API for managing firmware keys

### PR DESCRIPTION
**Pull request by mobileoverlord**
This PR adds routes for managing firmware keys

* get `/keys` - list all keys for the org
* post `/keys` - create a new key
* delete `/keys/:name` - delete a key by name
* get `/keys/:name` - show information about a single key

### CODE REVIEW

File: apps/nerves_hub_api/test/nerves_hub_api_web/controllers/key_controller.test.exs [hunk 1]
```diff
@@ -0,0 +1,51 @@
+defmodule NervesHubAPIWeb.KeyControllerTest do
```
Comment (jlmcgehee21) [inline apps/nerves_hub_api/test/nerves_hub_api_web/controllers/key_controller.test.exs]: our pattern for naming tests has been `_test.exs`, this file is named `.test.exs`

Comment (mobileoverlord) [reply apps/nerves_hub_api/test/nerves_hub_api_web/controllers/key_controller.test.exs]: whoops, good catch

**Comment by mobileoverlord** [comment]
@jlmcgehee21 There is a failing test I am getting at `test/nerves_hub_www_web/controllers/org_key_controller_test.exs:75` Its probably going to fail on CI, Would you be able to take a peek at it? I've been trying to fiddle with it but only led to getting frustrated.

**Comment by mobileoverlord** [comment]
Thanks :)

**Pull request merged**